src/main/resources/l10n/Menu_de.properties

```diff
@@ -211,3 +211,5 @@ Disable_highlight_groups_matching_entries=Keine_Gruppen_hervorheben
 Online_help_forum=Online-Hilfeforum
 
 Manage_protected_terms=
+
+BibTeX=
```

Comment:
I think adding these tests should be discussed in a separate Issue. @bruehldev s task here is to reenable the tests so that the JavaFX branch can be merged with the master.